test/oblivc/aes/aes.c

```diff
@@ -17,16 +17,17 @@ int main(int argc,char* argv[])
     if(argc<2) fprintf(stderr,"Port number missing\n");
     else if(argc<3) fprintf(stderr,"Party missing\n");
     else fprintf(stderr,"string missing\n");
-    fprintf(stderr,"Usage: %s <port> <1|2> <string>\n",argv[0]);
-    fprintf(stderr,"  Party 1 provides key in hex\n");
-    fprintf(stderr,"  Party 2 provides plaintext in hex\n");
+    fprintf(stderr,"Usage: %s <port> <--|remote_host> <string>\n",argv[0]);
```

Comment:
This is much better than the original way I think (no more recompiling to change the IP Address!), but I still feel like there's a lot of redundancy here among the various tests (which will only get worse as they're added). This isn't something I think you need to fix before merge (or that you should feel obligated to fix at all) - just something to note for the future. I have a partial solution to this problem in one of my repos, which may or may not be appropriate to adapt.